src/shared/report_op.c

```diff
@@ -472,13 +474,30 @@ void os_ReportdStart(report_filter *r_filter)
         }
 
         alerts_filtered++;
-        data_to_clean = (alert_data **) os_AddPtArray(al_data, (void **)data_to_clean);
+
+        /* If we have to dump the alerts */
+        if (r_filter->show_alerts) {
+            if (!msg_show_alerts) {
+                os_calloc(size_max_msg, sizeof(char), msg_show_alerts);
+                size_used_msg = snprintf(msg_show_alerts, size_max_msg, "Log dump:\n------------------------------------------------\n");
+            }
+
+            if ((size_max_msg - size_used_msg) < 200) {
```

Comment:
Why 200? Try not to hardcode static values, thanks.

Reply:
@bah07 and I talked about this problem. Right now I'm changing this to show only the last 10 alerts, so what I'm going to do is that, when `2*size_used_msg` is greater than `size_max_msg`, then I'll double the capacity of `msg_show_alerts` and update the value of `size_max_msg`.